lib/humidifier/aws_adapters/sdkv2.rb

```diff
@@ -10,9 +10,21 @@ def create_change_set(payload)
         try_valid { client.create_change_set(payload.create_change_set_params) }
       end
 
+      # Wrap the result from create_stack so it responds to :id in addition to
+      # :stack_id
+      class ResponseWrapper < SimpleDelegator
+        def id
+          stack_id
+        end
+      end
+
       # Create a change set if the stack exists, otherwise create the stack
       def deploy_change_set(payload)
-        exists?(payload) ? create_change_set(payload) : create(payload)
+        if exists?(payload)
+          create_change_set(payload)
+        else
+          create(payload).instance_eval { ResponseWrapper.new(self) }
```

Comment:
I'm find with the above change. I was going to suggest

```
response = create(payload)
response.singleton_class.send(:alias_method, :id, :stack_id)
response
```

but either way works.